lib/load.php

```diff
@@ -91,10 +101,19 @@ function gutenberg_is_experiment_enabled( $name ) {
 	require __DIR__ . '/compat/wordpress-6.4/html-api/class-wp-html-processor.php';
 }
 
+/*
+ * There was a bug in WordPress `trunk` during the 6.4 development cycle and that bug was
+ * introduced and fixed in the Tag Processor. Therefore, in case Gutenberg is running on
+ * a version of WordPress with the bug, it should run its own copy with the fix.
+ *
+ * This include needs to come after the dependencies above in case Gutenberg is running
+ * on WordPress 6.3 or older. The helper classes need to be loaded first.
+ */
+require __DIR__ . '/compat/wordpress-6.4/html-api/class-gutenberg-html-processor-6-4.php';
+
```

Comment:
See https://github.com/WordPress/gutenberg/pull/55703/files#r1376626781 -- the file seems essentially identical to `/compat/wordpress-6.4/html-api/class-wp-html-processor.php`.

If it was only WP `trunk` that was affected by that bug at some point -- but the bug has since been fixed -- I don't think we need to provide a workaround here; I don't think we typically make that sort of guarantee when people can simply update to a later `trunk`. 🤔

Reply:
see above reply. given that we had a `-6-3` version of the Tag Processor and given that we're going to continue to develop the HTML Processor, I feel like it's inevitable that we need this, and quickly so.

so if you think it's important to remove it I'm fine with that, but also if we remove it we'll probably need to add it in a week or whenever we have our first post-6-4 bug fix to the HTML Processor. happy to defer that until we need it, but again, I know this covers bugs that are possible _right now_ for folks running Gutenberg between 6.3 and 6.4.

Reply:
Yeah, I think I'd prefer removing `Gutenberg_HTML_Processor_6_4` (and having GB's `WP_HTML_Processor` subclass `Gutenberg_HTML_Tag_Processor_6_4` instead 😬 ), see [my other comment](https://github.com/WordPress/gutenberg/pull/55703#discussion_r1377384112).

If I'm not mistaken, we likely won't ever need a `Gutenberg_HTML_Processor_6_4`, as explained in that other comment 🤔 tl;dr:
- Any bugfixes to the HTML processor we'll also want in Core's `WP_HTML_Processor` in the 6.4 branch (for 6.4.x point releases), so we should be able to keep that file in sync with GB's file of the same name.
- Any new features for that HTML Processor that won't go into Core's 6.4 branch should go into Core's `trunk`, to be included in 6.5, and thus into `Gutenberg_HTML_Processor_6_5`.